tests/components/light/test_tradfri.py

```diff
@@ -35,20 +35,12 @@
             'brightness': 100
         }
     ],
-    # Brightness == 0
+    # Brightness == 1
     [
         {'can_set_dimmer': True},
-        {'brightness': 0},
+        {'brightness': 1},
         {
-            'brightness': 0
-        }
-    ],
-    # Brightness < 0
```

Comment:
This test was meaningless since brightness is clamped from 0..255 by the light schema